src/SFML/Graphics/Texture.cpp

```diff
@@ -44,6 +44,10 @@
 #include <climits>
 #include <cstring>
 
+#if defined(__GNUC__)
+    #pragma GCC diagnostic ignored "-Wduplicated-branches"
+#endif
+
```

Comment:
I think we can remove this in `master`. We have OpenGL ES jobs in v3 and this warning doesn't appear.